circus/tests/test_client.py

```diff
@@ -78,41 +77,51 @@ def test_client(self):
         yield self.stop_arbiter()
 
 
+TMP_FILE = './HOOK_CALLED'
+
+
 def long_hook(*args, **kw):
-    time.sleep(5)
+    with open(TMP_FILE, 'a'):
```

Comment:
Oh... When I applied the patch I remembered why I didn't use tempfile or a global var (for the same effect). The tempfile the hook sees, is different from the tempfile the test, as the module of the hook is resolved and loaded in a way that does not use the module of the test. Does this test pass from your side ?